.github/workflows/lint.yml

```diff
@@ -39,3 +39,10 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - uses: chartboost/ruff-action@v1
+
+  editorconfig-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: kasperhesthaven/setup-editorconfig-checker@v1.2.0
```

Comment:
Maybe we should just install it in our docker image and use directly. I get anxiety from using actions that literally [noone](https://github.com/search?q=kasperhesthaven%2Fsetup-editorconfig-checker&type=code) uses.

Or call directly, if we prefer to use ubuntu.
```
go install github.com/editorconfig-checker/editorconfig-checker/v3/cmd/editorconfig-checker@latest
```

Reply:
We were doing some changes in image recently, so not a problem to update it there. But I feel like it is better to use ubuntu if it is just one liner quick install.